tools/RepositoryManagement/DevHome.RepositoryManagement/ViewModels/RepositoryManagementItemViewModel.cs

```diff
@@ -74,6 +75,16 @@ public string Branch
 
     public bool HasAConfigurationFile { get; set; }
 
+    public string LatestCommitSHA { get; set; }
+
+    public string LatestCommitAuthor { get; set; }
+
+    public int MinutesSinceLatestCommit { get; set; }
```

Comment:
> So if I last committed something last year, we have some incomprehensible huge number.

Yes.

> If you have some UI that wants to show when the commit happened relative to now

RepositoryManagementMainPageViewModel.ConvertToLineItems does the time conversion and the conversion is ran every time the "Repository management" page is loaded.

However, this does mean, after a minute, the information is out-of-date.  Spoke with PMs and they are okay with having the page refresh only when the user navigates into the RepositorymanagementPage.

I'm going to leave this here because the scope of `RepositoryManagementItemViewModel` is scoped to the `Repository management` page.